### ISSUE TITLE
Is Pkl/K8s open sourced (yet)?

### ISSUE THREAD

**Issue by filleokus** [apple/pkl-k8s-examples#1]
Looks like the links to the package docs, like https://pkl-lang.org/package-docs/io.k8s/current/K8sResource/index.html and https://pkl-lang.org/package-docs/io.k8s.convert/index.html are not working.

When I tried to run the [basic example](https://github.com/apple/pkl-k8s-examples/blob/main/pkl/basic/README.adoc#basic-pklk8s-example) I got this error:

```
➜  basic git:(main) pkl eval -p input=../../yaml/frontend.yaml \
    -o frontend.pkl \
    applehub:io.k8s.convert:af50691d
–– Pkl Error ––
No module loader is registered for module `applehub:io.k8s.convert:af50691d`.
```

Is there something more that might need to be published publicly? :)

**Comment by kbwhodat** [comment]
Having the same issue. Do we have to register the module first prior to its use?

**Comment by holzensp** [comment]
Good catch. Thanks for filing the issue. Fixed in #3
